LightBlue.WebHost/Program.cs

```diff
@@ -152,9 +152,14 @@ private static string ObtainIisExpressConfigurationTemplate(WebHostArgs webHostA
 
         private static ProcessStartInfo BuildProcessStartInfo(WebHostArgs webHostArgs, string configurationFilePath)
         {
+            string envPath = Environment.GetEnvironmentVariable("ProgramW6432");
+            string path = webHostArgs.Use64Bit && !string.IsNullOrEmpty(envPath)
```

Comment:
This would be better as a test in the arguments parser than causes the process to fail fast rather than not work as expected.